bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router';
 import { defineMessages, injectIntl } from 'react-intl';
+import { makeCall } from '/imports/ui/services/api';
+import Button from '/imports/ui/components/button/component';
 import Modal from '/imports/ui/components/modal/fullscreen/component';
+import Auth from '/imports/ui/services/auth';
+import userListService from '../user-list/service';
```

Comment:
Oh, right, good catch. This part definitely has to be in the container, otherwise it's not gonna work.

But I'm still curious if we need to pass makeCall handlers through the createContainer, if they don't fetch any data.